tests/Tests/Api/PatientApiDisabled.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace OpenEMR\Tests\Api;
+
+use PHPUnit\Framework\TestCase;
+use OpenEMR\Tests\Api\ApiTestClient;
+use OpenEMR\Tests\Fixtures\FixtureManager;
+use GuzzleHttp\Exception\ClientException;
```

Comment:
Yep! Initially I configured Guzzle to throw an exception rather than return a response code. After I switched over to response codes, I missed this "import". Thanks!